ideascube/monitoring/tests/test_views.py

```diff
@@ -460,6 +609,18 @@ def test_can_return_loan(staffapp, user):
     assert not Loan.objects.due().count()
 
 
+def test_can_return_loan_from_serial(staffapp, user):
```

Comment:
Can we return a loan from serial if the loan had been made with the barcode? And the opposite?